src/proxy-ui-api/frontend/src/views/KeysAndCertificates/SignAndAuthKeys/KeysTable.vue

```diff
@@ -199,6 +220,28 @@ export default Vue.extend({
     importCert(hash: string): void {
       this.$emit('importCertByHash', hash);
     },
+    deleteCsr(req: TokenCertificateSigningRequest, key: Key): void {
+      this.confirmDeleteCsr = true;
+      this.selectedCsr = req;
+      this.selectedKey = key;
+    },
+    doDeleteCsr(): void {
```

Comment:
I'd try to come up with better method names as deleteCsr & doDeleteCsr sound basicly the same thing, but still the logic inside is quite different.